libs/ui-lib/lib/common/config/constants.ts

```diff
@@ -38,8 +38,10 @@ export const FEEDBACK_FORM_LINK =
 
 export const TECH_SUPPORT_LEVEL_LINK = 'https://access.redhat.com/support/offerings/techpreview';
 
-export const ENCRYPTING_DISK_DURING_INSTALLATION =
-  'https://docs.openshift.com/container-platform/4.7/installing/install_config/installing-customizing.html#installation-special-config-encrypt-disk_installing-customizing';
+export const getEncryptingDiskDuringInstallationDocsLink = (ocpVersion?: string) =>
+  `https://docs.openshift.com/container-platform/${
+    getShortOpenshiftVersion(ocpVersion) || '4.12'
```

Comment:
We probably should have a "defaultVersion" rather than having each doc by default pointing to a different version, or just the version  that the feature was first created for.

I think the best is always going to be the latest stable one, to show the more up to date documentation.